Contents/Code/__init__.py

```diff
@@ -105,8 +105,14 @@ def MainMenu():
         return oc
     else:
         try:
-            episodelistids = JSON.ObjectFromURL('http://' + Dict['private_ip'] + ':18080/plex/rec_ids', values=None,
-                                                headers={}, cacheTime=60)
+            recs = JSON.ObjectFromURL('http://' + Dict['private_ip'] + ':8885/recordings/airings', values=None, headers={}, cacheTime=60)
```

Comment:
all `/plex` routes are now deprecated - updated to the new `/recordings/airings` route.